lib/absinthe/logger.ex

```diff
@@ -0,0 +1,128 @@
+defmodule Absinthe.Logger do
+  @moduledoc """
+  Handles logging of various instrumentation events.
+
+  ## Variable filtering
+
+  Absinthe can filter out sensitive information like tokens and passwords
```

Comment:
Perhaps note that the definition of "filter out" is "replace with the string `[FILTERED]`"?